test/TestIncludeMissingFileRule.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import print_function
+
+import os
+import pytest
+
+from ansiblelint import Runner, RulesCollection
+
+PLAY_INCLUDING = '''
+- hosts: all
+  tasks:
+    - include: some_file.yml
+'''
+
+PLAY_INCLUDING_JINJA2 = '''
+- hosts: all
+  tasks:
+    - include: "{{ some_path }}/some_file.yml"
+'''
+
+PLAY_INCLUDING_NOQA = '''
+- hosts: all
+  tasks:
+    - include: some_file.yml # noqa 505
+'''
+
+PLAY_INCLUDED = '''
+- debug:
+    msg: 'was found & included'
+'''
+
+
+@pytest.fixture
+def linter_rules(scope="module"):
```

Comment:
It doesn't work like this. `scope` is an argument of the fixture decorator, not of the decorated function.
```suggestion
@pytest.fixture(scope='module')
def linter_rules():
```